CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,47 @@
+- Contributing to Nacto-care 🎓
+  Welcome to the EduWeb project! We're excited to have you contribute. Before you get started, please take a moment to review the guidelines for contributing.
```

Comment:
@aasthakourav20 Here it says EduWeb project. Please replace it with NactoCare and re-check the docs after pasting.